bugbug/bugzilla.py

```diff
@@ -290,13 +290,14 @@ def delete_bugs(match):
 def count_bugs(bug_query_params):
     bug_query_params["count_only"] = 1
 
-    r = utils.get_session("bugzilla").get(
-        "https://bugzilla.mozilla.org/rest/bug", params=bug_query_params
-    )
-    r.raise_for_status()
-    count = r.json()["bug_count"]
+    counts = [0]
+
+    def handler(bug):
+        counts[0] = bug["bug_count"]
```

Comment:
Using a list is a nice workaround. However, for the sake of consistency, I would suggest using a dictionary instead!

```suggestion
    data = {}

    def handler(bug):
        data["bug_count"] = bug["bug_count"]
```